python/pdstools/utils/cdh_utils.py

```diff
@@ -648,14 +680,7 @@ def _capitalize(fields: list) -> list:
 
 
 def _polarsCapitalize(df: pl.LazyFrame):
-    return df.rename(
-        dict(
-            zip(
-                df.columns,
-                _capitalize(df.columns),
-            )
-        )
-    )
+    return df.rename(dict(zip(df.columns, _capitalize(df.columns))))
```

Comment:
I think you have a slightly different linter than I do :) I don't mind it pulling this back to one line, but I do mind it pingponging back and forth between every single PR. Let's align on that soon